server/memphis_handlers_stations.go

```diff
@@ -1025,16 +898,7 @@ func (sh StationsHandler) RemoveStation(c *gin.Context) {
 		}
 	}
 
-	_, err := stationsCollection.UpdateMany(context.TODO(),
-		bson.M{
-			"name": bson.M{"$in": stationNames},
-			"$or": []interface{}{
-				bson.M{"is_deleted": false},
-				bson.M{"is_deleted": bson.M{"$exists": false}},
-			},
-		},
-		bson.M{"$set": bson.M{"is_deleted": true}},
-	)
+	err := db.DeleteStations(stationNames)
```

Comment:
DeleteStationsByNames. and check please this names are being converted to lower case

Reply:
converted to lower case in StationNameFromStr function